packages/ui-keyring/src/Base.ts

```diff
@@ -18,32 +18,32 @@ import { BrowserStore } from './stores';
 import { MAX_PASS_LEN } from './defaults';
 
 export default class Base {
-  private _accounts: AddressSubject;
-  private _addresses: AddressSubject;
-  private _contracts: AddressSubject;
+  private _account: AddressSubject;
+  private _address: AddressSubject;
+  private _contract: AddressSubject;
   private _keyring?: KeyringInstance;
   private _prefix?: Prefix;
   protected _genesisHash?: string;
   protected _store: KeyringStore;
 
   constructor () {
-    this._accounts = accounts;
-    this._addresses = addresses;
-    this._contracts = contracts;
+    this._account = accounts;
+    this._address = addresses;
+    this._contract = contracts;
     this._keyring = undefined;
     this._store = null as any;
   }
 
-  get accounts () {
-    return this._accounts;
+  get account () {
```

Comment:
Not sure it is worth breaking the external interface for this PR. So would forgo the renames on at least the getters.